packages/documentation/pages/usage/components/inline-edit-2.vue

```diff
@@ -0,0 +1,84 @@
+<template lang="md">
+<ComponentInfo v-bind="{ component }" />
+
+## Basic
+
+<div class="element-example white">
+	<KtInlineEdit2
+		label="Instructions"
+		placeholder="Click to edit"
+		value="my instructions"
+	/>
+</div>
+
+```html
+<KtInlineEdit2
+	label="Instructions"
+	placeholder="Click to edit"
+	value="my instructions"
+/>
+```
+
+## Multiline
+
+<div class="element-example white">
+	<KtInlineEdit2
+		:isMultiLine="true"
+		label="Instructions"
+		placeholder="Edit instructions"
+		value="my instructions"
+	/>
+</div>
+
+```html
+<KtInlineEdit2
+	:isMultiLine="true"
+	label="Instructions"
+	placeholder="Edit instructions"
+	value="my instructions"
+/>
+```
+
+## Usage
+
+### Attributes
+
+| Attribute     | Description                  | Type      | Accepted values | Default           |
+| :------------ | :--------------------------- | :-------- | :-------------- | :---------------- |
+| `isMultiLine` | allow input enter multilines | `Boolean` | —               | `false`           |
+| `label`       | label text                   | `String`  | —               | —                 |
+| `placeholder` | value to display             | `String`  | —               | `"Click to edit"` |
+
+### Input Events
+
+| Event Name | Description                       | Parameters       |
+| :--------- | :-------------------------------- | :--------------- |
+| `change`   | triggers when click outside input | `(event: Event)` |
+| `input`    | triggers when input value changes | `(event: Event)` |
+
+### Button Events
+
+| Event Name | Description                          | Parameters       |
+| :--------- | :----------------------------------- | :--------------- |
+| `confirm`  | triggers when confirm button clicked | --               |
+| `dismiss`  | triggers when dismiss button clicked | `(event: Event)` |
```

Comment:
why does `confirm` not have Paramters compared to `dismiss`

Reply:
because we do not need event in `handleConfirm`.
i would say that its not needed also in `handleDismiss`

Reply:
that's my point, it's equally unneeded :D